src/releaser/hexagon/ports/git_reader.py

```diff
@@ -33,3 +39,9 @@ def read_last_commit_message(self, depth: int, filter: str | None) -> str | None
                     return commit_msg
             return None
         return history[0]
+
+    def current_reference_matches(self, ref: list[str]) -> bool:
+        """Check if the current reference matches the given ref."""
+        if not ref:
+            return True
+        return self.read_current_branch() in ref
```

Comment:
This is a minimal effort: we don't support `*` character in branch name. For me it's ok